config/development.js

```diff
@@ -25,7 +26,7 @@ module.exports = {
   },
 
   cors: {
-    allowedOrigins: /(\.realdevsquad\.com$)|(localhost)/, // Allow *.realdevsquad.com and localhost for non-production envs
+    allowedOrigins: /(https:\/\/(.*\.)*realdevsquad\.com$)|(localhost)/, // Allow *.realdevsquad.com and localhost for non-production envs
```

Comment:
> Allow *.realdevsquad.com and localhost for non-production envs

The comment is slightly incorrect.

We also allow `realdevsquad.com` (without the subdomain), correct?

Reply:
Done